identityClient/api.go

```diff
@@ -177,14 +177,16 @@ type InitiateUserChallengeResponse struct {
 }
 
 type InitiateRegisterDeviceRequest struct {
-	TempPublicKey string `json:"temporary_public_key"`
+	TempPublicKey           string `json:"temporary_public_key"`
+	TempPublicEncryptionKey string `json:"temporary_public_encryption_key"`
```

Comment:
I realize this is adding to an existing construct, but typically when public signing and encryption key are sent to our API the encryption public key is referred to as 'public key' and the singing public key is referred to as 'signing key'. This flips that on it's head a little, but for consistency it may be worth considering the internal naming even if the JSON keys are different. A little easier to quickly understand what's happening for those familiar with our code.

Reply:
Yeah its a lack of foresight by me unfortunately, keeping consistent naming would require 3 more changes in frontend space (mobile apps and browser) as it's a breaking api change; since we won't have frontend support for totp at this time I don't think it a change is necessary and can come with those frontend updates.